### PR TITLE
how to enable Resource Providers that are disabled by default

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

File: content/en/docs/languages/java/automatic/configuration.md [hunk 1]
```diff
@@ -30,6 +30,23 @@ specific portions of the SDK & agent:
 - [Span limits](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#span-limits)
 - [Using SPI to further configure the SDK](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#customizing-the-opentelemetry-sdk)
 
+### Enable Resource Providers that are disabled by default
```
Comment (theletterf) [inline content/en/docs/languages/java/automatic/configuration.md:33]: Do "Resource Providers" need to be capitalized here?

**Review by theletterf — APPROVED** [review]

**Comment by theletterf** [comment]
Invoking @open-telemetry/java-maintainers here — thanks!

File: content/en/docs/languages/java/automatic/configuration.md (hunk 1 above)
Comment (zeitlinger) [reply content/en/docs/languages/java/automatic/configuration.md:33]: the link above has "OpenTelemetry Resource and service name" - so it's consistent

Comment (svrnm) [reply content/en/docs/languages/java/automatic/configuration.md:33]: then probably the link above should be updated to resource? ;-)

Comment (zeitlinger) [reply content/en/docs/languages/java/automatic/configuration.md:33]: I hope I got the one you mean :smile:

**Review by jeanbisutti — APPROVED** [review]

File: content/en/docs/languages/java/automatic/configuration.md (hunk 1 above)
Comment (cartermp) [reply content/en/docs/languages/java/automatic/configuration.md:33]: We can follow up with a capitalization pass if we like later. I suspect there's several things that would be changed across many docs.

**Review by cartermp — APPROVED** [review]

**Pull request merged**